docs: clarify what core features are (#19842)
microsoft/playwright: docs/src/languages.md

title: "Supported languages"
---

> Playwright is available in multiple languages that share the same underlying implementation. All core features (those used for automation of standard Web APIs available in all browsers) are supported in all languages. Pick the language based on your experience, familiarity with its testing ecosystem and your project constraints. For the best experience pick the test runner that we recommend for each language.

Reviewer: ```suggestion
Playwright is available in multiple languages that share the same underlying implementation. All core features for automating the browser are supported in all languages, while testing ecosystem integration is different. Pick the language based on your experience, familiarity with its testing ecosystem and your project constraints. For the best experience pick the test runner that we recommend for each language.
```
